packages/utils/src/helpers.ts

```diff
@@ -298,15 +330,27 @@ export const standardizePhases = phases =>
     return standardizedPhase;
   });
 
-const getInstallScriptArgs = ({ isHosted, isPreRelease, hasMonitor }) => {
+interface DebConfigurationProps {
+  deviceType?: string;
+  hasMonitor: boolean;
+  ipAddress?: string;
+  isDemoMode: boolean;
+  isHosted: boolean;
```

Comment:
do you think it would make sense to add member ordering rules to eslint?

Reply:
yeah, I was tempted to do that, but we rely on the order of some objects in the UI code, where we then iterate over the object entries... but since the iteration order afaik isn't guaranteed, we could add the rule and either ignore the places with expected order or make the iteration rely on a stable index 👌 => will do this in a follow up PR though

Reply:
https://github.com/NorthernTechHQ/nt-gui/pull/174 turns out the ts rule only is for ts things 😆 - so the regular object sorting order can be taken on a later date 👍